source/common/http/filter/buffer_filter.h

```diff
@@ -51,6 +51,7 @@ class BufferFilter : public StreamDecoderFilter {
   static BufferFilterStats generateStats(const std::string& prefix, Stats::Scope& scope);
 
   // Http::StreamFilterBase
+  void setBufferLimit(uint32_t) override {} // Buffer filter uses its own configured limits.
```

Comment:
Mostly thought exercise: Could we actually allow filter to set buffer limit and push 413 to connection manager? This would make this filter trivial in that all it does is provide config to set buffer limit to min(configured_limit, stream_limit).